src/includes/Exports-Include.hs

```diff
@@ -27,8 +28,11 @@
     -- line, while functions like @cursorUp@ and @cursorDown@ keep the column
     -- the same.
     --
-    -- Also keep in mind that @*Line@ functions are not as portable. See
-    -- <https://github.com/UnkindPartition/ansi-terminal/issues/10> for the details.
+    -- These code sequences are part of ECMA-48 standard, but some obsolete terminals,
+    -- foremost <https://en.wikipedia.org/wiki/ANSI.SYS ANSI.SYS>, do not support them.
+    -- Replacing them by more fine-grained calls, e. g., @cursorUp n >> setCursorColumn 0@
+    -- instead of @cursorUpLine n@, does not help, because ANSI.SYS does not support
+    -- @setCursorColumn@ as well.
```

Comment:
These functions are emulated on legacy Windows, so portability should not be an issue on Windows (legacy or modern).